cloud/scope/cluster.go

```diff
@@ -152,11 +152,18 @@ func (s *ClusterScope) LBSpecs() []azure.LBSpec {
 	return specs
 }
 
-// RouteTableSpecs returns the node route table(s)
+// RouteTableSpecs returns the node route table
 func (s *ClusterScope) RouteTableSpecs() []azure.RouteTableSpec {
-	return []azure.RouteTableSpec{{
-		Name: s.RouteTable().Name,
-	}}
+	return []azure.RouteTableSpec{
+		{
+			Name:   s.ControlPlaneRouteTable().Name,
```

Comment:
we should only add each routeTable if its name != ""

Reply:
only the control plane one? or both, we need to have the node route table, right?